Content.Server/Atmos/Piping/Unary/Components/GasThermoMachineComponent.cs

```diff
@@ -15,32 +12,24 @@ public sealed partial class GasThermoMachineComponent : Component
         ///     Current electrical power consumption, in watts. Increasing power increases the ability of the
         ///     thermomachine to heat or cool air.
         /// </summary>
-        [ViewVariables(VVAccess.ReadWrite)]
-        public float HeatCapacity = 10000;
-
-        /// <summary>
-        ///     Base heat capacity of the device. Actual heat capacity is calculated by taking this number and doubling
-        ///     it for every matter bin quality tier above one.
-        /// </summary>
-        [DataField("baseHeatCapacity")]
-        public float BaseHeatCapacity = 5000;
+        [DataField, ViewVariables(VVAccess.ReadWrite)]
+        public float HeatCapacity = 5000;
 
-        [DataField("targetTemperature")]
-        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField, ViewVariables(VVAccess.ReadWrite)]
         public float TargetTemperature = Atmospherics.T20C;
 
         /// <summary>
         ///     Tolerance for temperature setpoint hysteresis.
         /// </summary>
-        [ViewVariables(VVAccess.ReadOnly)]
+        [DataField, ViewVariables(VVAccess.ReadOnly)]
         public float TemperatureTolerance = 2f;
 
         /// <summary>
         ///     Implements setpoint hysteresis to prevent heater from rapidly cycling on and off at setpoint.
         ///     If true, add Sign(Cp)*TemperatureTolerance to the temperature setpoint.
         /// </summary>
-        [ViewVariables(VVAccess.ReadOnly)]
-        public bool HysteresisState = false;
+        [DataField, ViewVariables(VVAccess.ReadOnly)]
```

Comment:
DataField doesn't make sense here

Reply:
Doesn't it need to be serialized for mapsaving to work correctly?

It looks like it's used to store state across ticks.

Reply:
yes, it's a state variable, but didn't think of mapsaving

Reply:
and persist probably